contracts/interfaces/IAsset.sol

```diff
@@ -105,9 +116,7 @@ interface ICollateral is IAsset {
 
     /// @return {target/ref} Quantity of whole target units per whole reference unit in the peg
     function targetPerRef() external view returns (uint192);
-}
 
-interface TestICollateral is ICollateral, TestIAsset {
```

Comment:
Same thing for `ICollateral`. Not every collateral will have a `delayUntilDefault()` or `whenDefault()`..

Reply:
which ones wouldn't? I feel like theyre standard values for soft => hard defaults?

Reply:
Not all tokens have a bridge component that require a soft default check. We ended up by laziness keeping around a `delayUntilDefault` for `SelfReferentialCollateral`, but it's not a necessity. When other protocol teams develop plugins they should be free to not have this parameter. 

And for `whenDefault()`, that just comes out of inheriting from our parent classes, which isn't a necessity. `status()` is the real thing; `whenDefault()` just happened to come along for the ride